book-example/src/cli/build.md

```diff
@@ -35,5 +35,6 @@ not specified it will default to the value of the `build.build-dir` key in
 
 -------------------
 
-***Note:*** *Make sure to run the build command in the root directory and not in
-the source directory*
+***Note:*** *The build command copies all files (excluding files with `.md` extension) from the source directory
+into the build directory. If the build directory is under the source directory it is not copied as this would
```

Comment:
I don't think this second sentence is necessary.  I don't think there is a need to call out all the incorrect behaviors it tries to avoid.